include/aspect/simulator_access.h

```diff
@@ -781,14 +781,32 @@ namespace aspect
        * constraints for the time step we are currently solving.
        */
       const ConstraintMatrix &
-      get_current_constraints() const;
+      get_current_constraints () const;
 
       /**
        * Return whether or not the current object has been initialized by providing it with
        * a pointer to a Simulator class object.
```

Comment:
can you add a comment that this means that the initialization started? Maybe we should rename this function?

Reply:
The problem is that the `simulator_is_initialized()` function talks about the initialization of the `SimulatorAccess` object, whereas the new function talks about the initialization state of the `Simulator` object. I'm of course open to other names.